DESCRIPTION

```diff
@@ -47,7 +47,7 @@ Suggests:
 VignetteBuilder: 
     knitr
 Remotes:
-    bahadzie/numberize,
+    epiverse-trace/numberize,
     epiverse-trace/linelist
```

Comment:
Does the CRAN version not have the required functionality? If not, it would be useful to find out from @Bisaloo if the upcoming CRAN update will contain it so this dependency can be moved to `Imports`

Reply:
Thanks for the catch! Yes, it should be alright to use the CRAN linelist version since linelist 1.0.0

Reply:
@bahadzie - thanks for raising this

@Bisaloo - Thanks for the clarification

I will update this accordingly.